browser/brave_tab_helpers.cc

```diff
@@ -133,6 +134,8 @@ void AttachTabHelpers(content::WebContents* web_contents) {
 
   brave_ads::AdsTabHelper::CreateForWebContents(web_contents);
 
+  psst::PsstTabHelper::CreateForWebContents(web_contents);
+
```

Comment:
I don't know why we have all these extra spaces, but let's leave them out here. Also I would put this after `SearchResultAdTabHelper` so it's not between two brave_ads tab helpers